### ISSUE TITLE
Whole Word Masking Models update

### ISSUE THREAD

**Issue by frankxu2004** [huggingface/pytorch-transformers#659]
Recently Google updated their TF implementation (`https://github.com/google-research/bert`) with Whole Word Masking Models that masks whole random word instead of just random wordpieces, which results in a performance gain. 

Just wondering if this will be implemented here?

Thanks.

**Comment by thomwolf** [comment]
It's not yet but thanks for the pointer, we can probably add it fairly easily. I'll have a look.

**Comment by lirongyuan** [comment]
This would be very helpful!

**Comment by bkahn-github** [comment]
Hi, 

I converted the cased and uncased whole-word-masking models using the command line tool. If you're interested in adding these to the repository, I've uploaded them to [this](https://www.kaggle.com/bkkaggle/bert-large-whole-word-masking) kaggle dataset.

**Comment by sleepinyourhat** [comment]
Is this resolved? These seem to be available at head, and I don't see anything immediately wrong when I try them...

**Comment by thomwolf** [comment]
Yes they are working fine, I've added them to master last week.
They will be advertised in the next release.
When fine-tuned with run_squad they give decent results: `exact_match: 86.91, f1: 93.15`.
I've included a version fine-tuned on SQuAD as well.

**Comment by jshin49** [comment]
Does Whole Word Masking support bert base as well?
